src/assets/stylesheets/ui-root.scss

```diff
@@ -70,6 +70,10 @@
 
 .ui-interactive {
   pointer-events: auto;
+  -moz-user-select: none;
```

Comment:
I was gunna suggest that we dont need prefixes because this is older tahn sin, but apparently we still do. Oh CSS.